src/main/resources/static/styles.css

```diff
@@ -55,18 +55,14 @@
 
 }
 
-.gardens {
-    background-color: #cae8d5;
-}
-
 .card {
     border: none;
     margin-top: 3rem;
     margin-bottom: 3rem;
 }
 
 .card-img-top {
-    height: 20vw;
+    height: 25vw;
```

Comment:
For height I suggest using vh (viewport height). Using vh actually solves the problem of the image getting very small on mobile screen sizes. The image will always be 25% of the viewport height.

Reply:
I agree, I just tried vh instead and it does take care of the problem that Heather brought up about cropped pictures.  However, I would recommend setting it at 40vh.  25vh makes the pictures too small IMO.

Reply:
I agree with both points, changes made and they look great.

Reply:
Good to know about vh! good tips!